spec/factories/admin_user.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+    factory :admin_user do
+      email "user@kicks4love.com"
+      username "Robin Lou"
+      password "grandcherokee"

```

Comment:
I need some explanation on what is going on here 😅

Reply:
🐴